tests/test_actions.py

```diff
@@ -83,11 +89,17 @@ def test_falls_back_to_pdb(self, mocked_pdb, mocked_breakpoint, Tester):
 
         mocked_pdb.set_trace.assert_called_once()
 
+    def test_describe(self):
+        assert Debug().describe() == "Assume direct control."
+
 
 class TestEventually:
     def get_mock_action(self, **kwargs):
-        mock_action = mock.Mock()
-        mock_action.perform_as = mock.Mock(**kwargs)
+        mock_action = mock.create_autospec(Action, _name="fakeaction", instance=True)
+        # TODO: figure out why using create_autospec on the method causes exception to
+        #  occur when calling perform_as()
+        # mock_action.perform_as = mock.create_autospec(Action.perform_as, **kwargs)
```

Comment:
What exception is raised?

Reply:
`TypeError: missing a required argument: 'the_actor'`

Reply:
Ah, that's probably because you're doing `Action.perform_as` instead of `Action().perform_as`? `perform_as` is an instance method, so i wonder what happens when you provide it like a static method.